index.js

```diff
@@ -23,8 +29,8 @@ db.sequelize
   .sync()
   .then(startServer)
   .catch((err) => {
-      if (err) debug('An error occured %j', err);
-      else debug('Database synchronized');
+      if (err) logger.debug('An error occured', err);
```

Comment:
NB: Doesn't look like you've configured winston to error formatting ==> `err = new Error()` object will not actually log.